app/features/config/SelectWowDir.tsx

```diff
@@ -17,6 +17,9 @@ function getDefaultPath() {
   if (os === 'win32') {
     return 'C:\\Program Files (x86)\\World of Warcraft\\_retail_';
   }
+  if (os === 'linux') {
+    return require('os').homedir();
```

Comment:
Use import at the top of the file. Usually you use require in nodejs (this is called commonjs formatting system), but Electron is quite weird in that node in a ES6 module environment. So use import unless you need to import it dynamically.